src/main/java/com/olgaboiar/mint/ServerRoutes.yaml

```diff
@@ -0,0 +1,54 @@
+---
+path: /simple_get
+methodhandlers:
+  - method: GET
+    handler: ROUTE_HANDLER
+  - method: HEAD
+    handler: HEAD_HANDLER
+---
+path: /method_options
+methodhandlers:
+  - method: GET
+    handler: ROUTE_HANDLER
+  - method: HEAD
+    handler: HEAD_HANDLER
+  - method: OPTIONS
+    handler: OPTIONS_HANDLER
+---
+path: /get_with_body
+methodhandlers:
+  - method: HEAD
+    handler: HEAD_HANDLER
+  - method: OPTIONS
+    handler: OPTIONS_HANDLER
+---
+path: /method_options2
+methodhandlers:
+  - method: GET
+    handler: ROUTE_HANDLER
+  - method: HEAD
+    handler: HEAD_HANDLER
+  - method: OPTIONS
+    handler: OPTIONS_HANDLER
+  - method: PUT
+    handler: NOT_ALLOWED_HANDLER
+  - method: POST
+    handler: NOT_ALLOWED_HANDLER
```

Comment:
Needing to explicitly denote methods as not allowed for a resource can get cumbersome to maintain, I feel like the resource existing (`/method_options2`) but the method not being bound can determine a `NOT_ALLOWED_HANDLER`'s response, and might makes this feature a little bit more robust. Let me know if you've got any questions or want to pair.

Reply:
fixed in https://github.com/olgaboiar/MintHTTPServer/pull/10/commits/7388c9a216a53c50fce8bbeab0bdef5788072485